Pipfile

```diff
@@ -12,7 +12,6 @@ pyodbc = "==4.0.27"
 pyorient = "==1.5.5"
 redis = ">=3.2.1,<4"
 redisgraph = ">=1.7,<1.9"
-
```

Comment:
Did the linter remove the newlines for you? I find that they help with readability

Reply:
I used `pipenv install` instead of manually editing the pipfile

Reply:
Gotcha - can you put them back 😇 ?